k8s-apply/src/Makefile

```diff
@@ -0,0 +1,7 @@
+DOCKER ?= docker
+
+build-docker:
+	$(DOCKER) build -t signadot/k8s-apply:latest .
```

Comment:
Do we want to version the image?

Reply:
It's done here: https://github.com/signadot/plugins/pull/19/commits/78a29d8005f3c32f3af0f63308f2863e7e2dc592.